finality-aleph/src/new_network/data_network/aleph_network.rs

```diff
@@ -0,0 +1,44 @@
+use crate::{
+    crypto::Signature,
+    data_io::{AlephDataFor, AlephNetworkMessage},
+    new_network::data_network::split::DataNetwork,
+    Hasher,
+};
+use aleph_bft::SignatureSet;
+use log::error;
+use sp_api::BlockT;
+
+pub(crate) type AlephNetworkData<B> =
+    aleph_bft::NetworkData<Hasher, AlephDataFor<B>, Signature, SignatureSet<Signature>>;
+
+impl<B: BlockT> AlephNetworkMessage<B> for AlephNetworkData<B> {
+    fn included_blocks(&self) -> Vec<AlephDataFor<B>> {
+        self.included_data()
+    }
+}
+
+pub(crate) struct AlephNetwork<B: BlockT> {
+    inner: DataNetwork<AlephNetworkData<B>>,
+}
+
+impl<B: BlockT> AlephNetwork<B> {
+    pub(crate) fn new(inner: DataNetwork<AlephNetworkData<B>>) -> Self {
+        AlephNetwork { inner }
+    }
+}
+
+#[async_trait::async_trait]
+impl<B: BlockT> aleph_bft::Network<Hasher, AlephDataFor<B>, Signature, SignatureSet<Signature>>
+    for AlephNetwork<B>
+{
+    fn send(&self, data: AlephNetworkData<B>, recipient: aleph_bft::Recipient) {
+        let recipient = recipient.into();
+        if self.inner.send(data, recipient).is_err() {
+            error!(target: "afa", "error sending a message to {:?}", recipient);
```

Comment:
This should probably be a `warn`, this means this specific session is broken, but the program can technically continue operating in this degraded state.

Reply:
Done